### PR TITLE
Feat: Map corrections

### LINKED ISSUE #2097: MAP: loadLuaMapAction Error Startup Server
Sem título

### PR DESCRIPTION
**Pull request by dudantas**
Minor corrections in respawns and corrections in the general map, added items where none existed, corrected some houses and removed items that were not needed
Fixed error in loadLuaMapAction (variable with wrong name)

Registered new doors, ids: 
38467
38468
38469
38470
38471
38472
39056
30057
39058
39059

Fixed wooden ladder from id 35964 and 35965

Thanks for @dex-89

### CODE REVIEW

**Review by dex-89 — APPROVED** [review]

**Comment by dex-89** [comment]
-  Position: `32363, 32504, 7` is **trap**. Wooden ladder works as stairs.
-  Position: `32358, 32503, 7`, `32364, 32501, 7`, `32363, 32502, 6` doors no work
-  Position: `32474, 32498, 7` and `32475, 32498, 7` enter a ship no work
-  Position: `32453, 32507, 7` teleport -> probably to quest, so 👍 
-  Position: `32392, 32496, 7` stone wall.. where is the entrace?
-  Position: `32400, 32497, 7` door no work and `32400, 32497, 7` Wooden ladder works as stairs. After you descend, you are in the wall
-  Position: `32406, 32494, 7`, `32413, 32501, 7`, `32421, 32502, 7` next doors... -> doors no work at Bounac

File: data/world/otservbr-house.xml [hunk 1]
```diff
@@ -964,23 +958,30 @@
...
-	<house name="Pilchard Bin 5" houseid="3674" entryx="32280" entryy="31148" entryz="7" rent="80000" townid="16" size="8" />
-	<house name="Pilchard Bin 6" houseid="3675" entryx="32281" entryy="31164" entryz="6" rent="25000" townid="16" size="7" />
-	<house name="Pilchard Bin 7" houseid="3676" entryx="32281" entryy="31160" entryz="6" rent="25000" townid="16" size="7" />
-	<house name="Pilchard Bin 8" houseid="3677" entryx="32281" entryy="31156" entryz="6" rent="25000" townid="16" size="5" />
-	<house name="Pilchard Bin 9" houseid="3678" entryx="32281" entryy="31152" entryz="6" rent="50000" townid="16" size="7" />
-	<house name="Pilchard Bin 10" houseid="3679" entryx="32281" entryy="31148" entryz="6" rent="50000" townid="16" size="7" />
-	<house name="Mammoth House" houseid="3681" entryx="32234" entryy="31227" entryz="7" rent="300000" townid="16" size="155" />
+	<house name="Lion's Head Reef" houseid="3660" entryx="32365" entryy="32501" entryz="7" rent="300000" townid="25" size="152" />
+	<house name="Shadow Caves 1" houseid="3661" entryx="32654" entryy="31672" entryz="8" rent="5000" townid="5" size="17" />
```
Comment (dex-89) [inline data/world/otservbr-house.xml:962]: **50000**

File: data/world/otservbr-house.xml [hunk 2]
```diff
@@ -964,23 +958,30 @@
...
+	<house name="Shadow Caves 2" houseid="3662" entryx="32656" entryy="31672" entryz="8" rent="5000" townid="5" size="20" />
```
Comment (dex-89) [inline data/world/otservbr-house.xml:963]: **50000**

File: data/world/otservbr-house.xml [hunk 3]
```diff
@@ -964,23 +958,30 @@
...
+	<house name="Shadow Caves 3" houseid="3663" entryx="32654" entryy="31664" entryz="9" rent="10000" townid="5" size="38" />
```
Comment (dex-89) [inline data/world/otservbr-house.xml:964]: **100000**

File: data/world/otservbr-house.xml [hunk 4]
```diff
@@ -964,23 +958,30 @@
...
+	<house name="Shadow Caves 4" houseid="3664" entryx="32656" entryy="31664" entryz="9" rent="10000" townid="5" size="38" />
```
Comment (dex-89) [inline data/world/otservbr-house.xml:965]: **100000**

File: data/world/otservbr-house.xml [hunk 5]
```diff
@@ -964,23 +958,30 @@
...
+	<house name="Shadow Caves 5" houseid="3665" entryx="32654" entryy="31672" entryz="9" rent="10000" townid="5" size="38" />
```
Comment (dex-89) [inline data/world/otservbr-house.xml:966]: **100000**

File: data/world/otservbr-house.xml [hunk 6]
```diff
@@ -964,23 +958,30 @@
...
+	<house name="Shadow Caves 6" houseid="3666" entryx="32656" entryy="31672" entryz="9" rent="10000" townid="5" size="38" />
```
Comment (dex-89) [inline data/world/otservbr-house.xml:967]: **100000**

File: data/world/otservbr-house.xml [hunk 7]
```diff
@@ -964,23 +958,30 @@
...
-	<house name="Pilchard Bin 4" houseid="3672" entryx="32280" entryy="31152" entryz="7" rent="50000" townid="16" size="10" />
-	<house name="Pilchard Bin 5" houseid="3674" entryx="32280" entryy="31148" entryz="7" rent="80000" townid="16" size="8" />
-	<house name="Pilchard Bin 6" houseid="3675" entryx="32281" entryy="31164" entryz="6" rent="25000" townid="16" size="7" />
-	<house name="Pilchard Bin 7" houseid="3676" entryx="32281" entryy="31160" entryz="6" rent="25000" townid="16" size="7" />
-	<house name="Pilchard Bin 8" houseid="3677" entryx="32281" entryy="31156" entryz="6" rent="25000" townid="16" size="5" />
-	<house name="Pilchard Bin 9" houseid="3678" entryx="32281" entryy="31152" entryz="6" rent="50000" townid="16" size="7" />
-	<house name="Pilchard Bin 10" houseid="3679" entryx="32281" entryy="31148" entryz="6" rent="50000" townid="16" size="7" />
-	<house name="Mammoth House" houseid="3681" entryx="32234" entryy="31227" entryz="7" rent="300000" townid="16" size="155" />
+	<house name="Lion's Head Reef" houseid="3660" entryx="32365" entryy="32501" entryz="7" rent="300000" townid="25" size="152" />
```
Comment (dex-89) [inline data/world/otservbr-house.xml]: **400000**
https://www.tibia.com/community/?subtopic=houses&page=view&houseid=63013&world=Secura

**Comment by gpedro** [comment]
houses.xml updated to add client attribute.

- - -

note:
missing 2 houses:
- Tunnel Gardens 2
- Harbour Place 4

**Comment by dex-89** [comment]
> houses.xml updated to add client attribute.
> 
> note:
> missing 2 houses:
> 
> * Tunnel Gardens 2
> * Harbour Place 4

I will check what has changed

**Review by dex-89 — DISMISSED** [review]
Houses (with this houseid) are not loading in RME

**Comment by dudantas** [comment]
> houses.xml updated to add client attribute.
> 
> note:
> missing 2 houses:
> 
> * Tunnel Gardens 2
> * Harbour Place 4

They are on the map... You need to find a way to add the client ids and the rme to identify this. The houses are linked to the map and not to the houses.xml (XML is only for storage)
Whenever any changes to the map are made, it will reset everything you do in houses.xml

**Comment by dudantas** [comment]
> * Position: `32363, 32504, 7` is **trap**. Wooden ladder works as stairs.
> * Position: `32358, 32503, 7`, `32364, 32501, 7`, `32363, 32502, 6` doors no work
> * Position: `32474, 32498, 7` and `32475, 32498, 7` enter a ship no work
> * Position: `32453, 32507, 7` teleport -> probably to quest, so 👍
> * Position: `32392, 32496, 7` stone wall.. where is the entrace?
> * Position: `32400, 32497, 7` door no work and `32400, 32497, 7` Wooden ladder works as stairs. After you descend, you are in the wall
> * Position: `32406, 32494, 7`, `32413, 32501, 7`, `32421, 32502, 7` next doors... -> doors no work at Bounac

Fixed, thanks

**Comment by Juanzin93** [comment]
wooden ladder changes 35964 35965 are wrong, in real tibia you stepIn on them

**Pull request merged**